samcli/lib/generated_sample_events/events.py

```diff
@@ -5,7 +5,7 @@
 import os
 import json
 import base64
-from requests.utils import quote as url_quote
+from urllib.parse import quote as url_quote
```

Comment:
following the import in `requests.utils`, the `quote` is actually imported from `urllib.parse` (for Python 3)